src/index.js

```diff
@@ -244,6 +276,31 @@ const processEventBridgeLogs = (EventBridgeLogRecords) => {
   EventBridgeLogRecords.log_type = 'aws_eventbridge';
 };
 
+/**
+ * This code process CodeCommit Logs
+ * @param {*} CodeCommitLogRecords 
+ */
+const processCodeCommitLogs = (CodeCommitLogRecords) => {
+  const ingestionTime = Date.now();
+  for (const record of CodeCommitLogRecords) {
+    record.ingest_timestamp = ingestionTime;
+    record.log_type = 'aws_codecommit';
+  }
+};
+
+/**
+ * This code process MSK Logs
+ * @param {*} MSKLogRecords 
+ */
+const processMSKLogs = (MSKLogRecords) => {
+  flattenMSKObject(MSKLogRecords);
+  for (const record of MSKLogRecords.Records) {
```

Comment:
replace this --- MSKLogRecords.Records
to  ---- MSKLogRecords

pass the records from the caller of this method.